mscore/musescore.h

```diff
@@ -449,6 +449,9 @@ class MuseScore : public QMainWindow, public MuseScoreCore {
 
       void updateViewModeCombo();
       void switchLayoutMode(LayoutMode);
+    
+    
```

Comment:
While in picky mode ;.): this is at least one linefeed too much, also some spaces.